packages/xstate-store/src/types.ts

```diff
@@ -365,14 +368,20 @@ export interface Readable<T> extends Subscribable<T> {
   get: () => T;
 }
 
-export interface Atom<T> extends Subscribable<T>, Readable<T> {
+interface BaseAtom<T> extends Subscribable<T>, Readable<T> {}
+
+export interface Atom<T> extends BaseAtom<T> {
```

Comment:
q: why `ReadonlyAtom` is a `Dependency` but an `Atom` isn't one? and similar question about `Subsciber`

Reply:
Oversight; fixed